__test__/integrationTests.test.ts

```diff
@@ -633,6 +633,40 @@ describe("Integration tests for AUTH routes:", () => {
             expect(updatedUser.status).toBe(401);
         });
     });
+
+    // Delete User route
+    // ------------------------------------------------------------------------
+    describe("deleteUser endpoint tests:", () => {
+        test("User is able to delete own account", async () => {
```

Comment:
I don't know how to test if someone could delete another users account, since its based on the session cookie the only way to delete another persons account is to get your hands on someone else's cookie and make a postman/http request using that cookie. Do you have any suggestions for how we could test that we cannot delete other people's accounts?

Reply:
Nope, that's a good point. And a good design. Without some crazy hacking skillz it won't happen so I would say no need to test. 👍